src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -41138,33 +41138,46 @@ static void Main()
                 //         x = x != y;
-                Diagnostic(ErrorCode.ERR_NoImplicitConv, "x != y").WithArguments("bool", "I1").WithLocation(25, 13)
+                Diagnostic(ErrorCode.ERR_FeatureInPreview, "x != y").WithArguments("static abstract members in interfaces").WithLocation(25, 13)
+                );
+
+            CreateCompilation(source1 + source2, options: TestOptions.DebugExe, targetFramework: TargetFramework.NetCoreApp,
+                              parseOptions: TestOptions.RegularPreview).VerifyDiagnostics(
+                // (4,31): error CS0567: Conversion, equality, or inequality operators declared in interfaces must be abstract
+                //     public static I1 operator ==(I1 x, I1 y)
+                Diagnostic(ErrorCode.ERR_InterfacesCantContainConversionOrEqualityOperators, "==").WithLocation(4, 31),
+                // (10,31): error CS0567: Conversion, equality, or inequality operators declared in interfaces must be abstract
+                //     public static I1 operator !=(I1 x, I1 y)
+                Diagnostic(ErrorCode.ERR_InterfacesCantContainConversionOrEqualityOperators, "!=").WithLocation(10, 31)
                 );
 
             CompilationReference compilationReference = compilation1.ToMetadataReference();
 
             var compilation2 = CreateCompilation(source2, new[] { compilationReference }, options: TestOptions.DebugExe,
-                                                 parseOptions: TestOptions.Regular, targetFramework: TargetFramework.NetCoreApp);
-            compilation2.VerifyDiagnostics(
-                // (9,13): error CS0029: Cannot implicitly convert type 'bool' to 'I1'
+                                                 parseOptions: TestOptions.RegularPreview, targetFramework: TargetFramework.NetCoreApp);
+            compilation2.VerifyDiagnostics();
+
+            CreateCompilation(source2, new[] { compilationReference }, options: TestOptions.DebugExe,
+                              parseOptions: TestOptions.Regular9, targetFramework: TargetFramework.NetCoreApp).VerifyDiagnostics(
+                // (9,13): error CS8652: The feature 'static abstract members in interfaces' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
                 //         x = x == y;
-                Diagnostic(ErrorCode.ERR_NoImplicitConv, "x == y").WithArguments("bool", "I1").WithLocation(9, 13),
-                // (10,13): error CS0029: Cannot implicitly convert type 'bool' to 'I1'
+                Diagnostic(ErrorCode.ERR_FeatureInPreview, "x == y").WithArguments("static abstract members in interfaces").WithLocation(9, 13),
+                // (10,13): error CS8652: The feature 'static abstract members in interfaces' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
```

Comment:
It feels a bit odd for this to be under "abstract static members in interfaces" if it's not abstract. Perhaps a separate feature flag?

Reply:
> It feels a bit odd for this to be under "abstract static members in interfaces" if it's not abstract. Perhaps a separate feature flag?

I am not sure it makes sense to add a separate feature for this scenario because there isn't a feature to declare non-abstract equality operators in interfaces, this isn't supported by the language. So, claming that you can enable something that isn't supported is probably going to be even more confusing.